src/actions/handleGithubComment.js

```diff
@@ -0,0 +1,50 @@
+
+const axios = require('axios');
+const config = require('../../.config')
+module.exports = ({task, conn}) => async function handleGithubComment(params) {
```

Comment:
Add a terst for this similar to https://github.com/mongoosejs/backend/blob/master/test/updateSubscriber.test.js